src/main/java/net/pms/formats/v2/AudioAttribute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * PS3 Media Server, for streaming any media to your PS3.
+ * PS3 Media Server, for streaming any medias to your PS3.
```

Comment:
Why not.
But as at first you said that it need to be corrected, and that you already corrected some; i just got confuse. I remember one of your PR with all of them corrected, though i don't where it is anymore. Anyways, that's not the end of the world ;-)